### DISCUSSION TITLE
제페토패키지 파일 등록 시, '월드 빌드 실패' 에러 로그 확인 방법

### DISCUSSION THREAD

**Discussion by Joyounwoo** [naverz/zepeto-studio-kor#565] [5. Q&A]
현재 월드 빌드 실패 발생 중입니다.
1. 유니티 로그인 된 제페토 계정과 상품 등록 계정은 동일한 계정입니다.
2. 월드 ID는 동일하고, 정규식에 이상 없습니다.
3. 유니티 상에선 정상적으로 동작합니다.

월드 빌드 실패 시 그 원인이나, 이유를 확인할 방법이 있을까요?
없다면 주로 어떤 원인으로 이와 같은 문제가 발생할까요?

<img width="1007" alt="스크린샷 2022-04-01 오후 1 30 29" src="https://user-images.githubusercontent.com/49576098/161197878-2f11b5f8-b8d5-4598-b20c-988ab8e1d3bc.png">

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요. 서버코드에 for문 if문 등을 사용할 때 중괄호가 없으면 위와 같이 나올수가 있습니다.
가이드 하단에 **서버 코드 작성 시 주의 사항**을 참고 부탁 드립니다.
https://studio.zepeto.me/kr/guides/multiplay

**Reply by Joyounwoo** [reply]
서버코드를 확인해보니 중괄호를 사용하지 않은 부분이 있었습니다.
수정 후 업로드하니 정상적으로 동작하네요.
답변 감사합니다!

**Reply by koldshine** [reply]
> 안녕하세요. 서버코드에 for문 if문 등을 사용할 때 중괄호가 없으면 위와 같이 나올수가 있습니다. 가이드 하단에 **서버 코드 작성 시 주의 사항**을 참고 부탁 드립니다. https://studio.zepeto.me/kr/guides/multiplay

저도 같은 증상인데요
혹시 [조건부 ? 값1 : 값2]와 같은 코드도 실패 원인이 될 수 있나요?
그리고 알려주신 링크에서 주의사항이 보이지 않습니다 ㅠ

**Reply by Zepeto-TechEvangelist** [reply]
안녕하세요, 서버 코드 중괄호 누락시 패키지가 정상적으로 업로드 되지 않던 현상은 1.3.0에서 개선되어서 가이드에서 빠졌습니다. https://github.com/naverz/zepeto-studio-kor/discussions/677
말씀하신 코드는 실패 원인이 아닙니다.
